lib/msal-angular/src/msal.service.ts

```diff
@@ -4,204 +4,78 @@
  */
 
 import { Inject, Injectable } from "@angular/core";
+import { Location } from "@angular/common";
 import {
-    UserAgentApplication,
-    Configuration,
-    AuthenticationParameters,
-    AuthResponse,
-    AuthError,
-    authResponseCallback,
-    errorReceivedCallback,
-    tokenReceivedCallback,
-    UrlUtils
-} from "msal";
-import { Router } from "@angular/router";
-import {BroadcastService} from "./broadcast.service";
-import { MSALError } from "./MSALError";
-import { MsalAngularConfiguration } from "./msal-angular.configuration";
-import { MSAL_CONFIG, MSAL_CONFIG_ANGULAR } from "./constants";
-
-import { Minimatch } from "minimatch";
-
-const buildMsalConfig = (config: Configuration) : Configuration => {
-    return {
-        ...config,
-        framework: {
-            ...config.framework,
-            isAngular: true
-        }
-    };
-};
+    IPublicClientApplication,
+    AccountInfo,
+    EndSessionRequest,
+    AuthorizationUrlRequest,
+    AuthenticationResult,
+    PopupRequest,
+    RedirectRequest,
+    SilentRequest
+} from "@azure/msal-browser";
+import { MSAL_INSTANCE } from "./constants";
+import { Observable, from } from "rxjs";
+
+interface IMsalService {
```

Comment:
This should be exported from this file and from `public_api.ts`.

Reply:
Done.